frontend/src/components/Application/Posts/views/PostEditor backup.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="container mt-2">
+    <h1 class="mb-3">Post Something</h1>
+    <v-form @submit.prevent="onSubmit">
+      <v-textarea
+        v-model="post_body"
+        class="form-control"
+        placeholder="What do you want to post?"
+        rows="3"
+        x-webkit-speech
```

Comment:
What is this?

Reply:
x-webkit-speech? something deprecated, I'll get rid of it.